modules/base/src/processors/volumesequencesource.cpp

```diff
@@ -92,14 +94,20 @@ std::optional<std::filesystem::path> getFirstFileInFolder(const std::filesystem:
 VolumeSequenceSource::VolumeSequenceSource(InviwoApplication* app)
     : Processor()
     , rf_{util::getDataReaderFactory(app)}
-    , outport_("data")
-    , inputType_("inputType", "Input type",
-                 {{"singlefile", "Single File", InputType::SingleFile},
-                  {"folder", "Folder", InputType::Folder}},
-                 1)
-    , file_("filename", "Volume file")
-    , folder_("folder", "Volume folder")
-    , filter_("filter_", "Filter", "*")
+    , outport_("data", "A sequence of volumes"_help)
+    , inputType_(
+          "inputType", "Input type",
+          "Select the input type, either select a single file to a 4D dataset or use a folder"_help,
+          {{"singlefile", "Single File", InputType::SingleFile},
+           {"folder", "Folder", InputType::Folder}},
+          1)
+    , file_("filename", "Volume file", "If using single file mode, the file to load"_help)
+    , folder_("folder", "Volume folder",
+              "If using folder mode, the folder to look for data sets in"_help)
+    , filter_(
```

Comment:
In the future, we should probably use a file pattern property here instead of a plain string.

Reply:
if someone cleans up those...